platform/windows/os_windows.cpp

```diff
@@ -2451,7 +2451,13 @@ Error OS_Windows::execute(const String &p_path, const List<String> &p_arguments,
 
 		for (const List<String>::Element *E = p_arguments.front(); E; E = E->next()) {
 
-			argss += String(" \"") + E->get() + "\"";
+			argss += " \"" + E->get() + "\"";
+		}
+
+		argss += "\"";
```

Comment:
Doesn't this add one `"` too many?

Reply:
Seems OK, last one closes `"` opened at line 2450.

```c++
argss = "\"\"" + p_path + "\"";

		for (const List<String>::Element *E = p_arguments.front(); E; E = E->next()) {
			argss += " \"" + E->get() + "\"";
		}

argss += "\"";
```

Full command in the one layer of `"` and path/each arg in the nested layer.

Reply:
Ah right, so it was bogus previously.